### ISSUE TITLE
Remove unused filter fallback on all css files

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/server#6408]
https://caniuse.com/#search=opacity
@nextcloud/designers 
Since we dropped ie8 support a while back, there should not be any filter fallback needed. Are you all okay with this?
This will be a simple cleanup! :)

```
~http/nextcloud-dev/core/css% grep -rnF 'filter:' | wc -l
43
```

**Comment by MorrisJobke** [comment]
Do it 😉

**Comment by skjnldsv** [comment]
Will do after #6407

**Comment by MorrisJobke** [comment]
Fix is in #6523
